main.nf

```diff
@@ -4,7 +4,7 @@
     nf-core/ampliseq
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Github : https://github.com/nf-core/ampliseq
-    Website: https://nf-co.re/ampliseq
+Website: https://nf-co.re/ampliseq
```

Comment:
Looks a bit out of line.

Reply:
I agree, thats the template update's fault, will change.